eng/Build.props

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup Condition="'$(Test)' == 'true'">
     <ProjectToBuild Include="$(RepoRoot)tests\GenerateScriptTests\GenerateScript.Tests.csproj" Test="$(Test)" />
+    <ProjectToBuild Include="$(RepoRoot)tests\ValidateSbrpTests\ValidateSbrp.Tests.csproj" Test="$(Test)" />
```

Comment:
We can simplify things by having just one test project (should probably be renamed to something generic like `SbrpTests`).

Reply:
Addressed in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/ce1a70dc071c6a5adce2b2d0ca6285031ee12258